src/docs/reference/templates.md

```diff
@@ -107,3 +104,54 @@ You can see all of your templates on your [Account's Templates page](https://rai
  width={3100}
  quality={80}
 />
+
+### Sharing your Templates
+
+After you create your template, you may want to share your work with the public and let others clone your project.
+
+#### Deploy on Railway Button
+
+Upon template creation, you are provided with the Template URL where your template can be found and deployed.  
+
+To complement your template, we also provide a `Deploy on Railway` button which you can include in your README or embed it into a website.
+![https://railway.app/button.svg](https://railway.app/button.svg)
+The button is located at [https://railway.app/button.svg](https://railway.app/button.svg).
+
+For examples and more information about the button, head over to our [Deploy On Railway Button](/deploy/deploy-on-railway-button) page.
+
+#### Kickback program
+
+If your published template is deployed into other users' projects, you are immediately eligible for a 25% kickback, in the form of Railway credits, of the usage cost incurred by those users.  That means if a user deploys your template, and the usage of the services cost the user $100, you could receive $25 in Railway credits.
+
+Important things to note:
```

Comment:
We should also add the minimum required usage for kickback. It is currently $4 (after discounts and platform fees) as the minimum kickback is $1. We can update it when we add fractional kickbacks.